pyproject.toml

```diff
@@ -9,12 +9,17 @@ license = { text = "LGPL-2.1-only" }
 dependencies = [
     "jinja2",
     "pydantic",
-    "pytest",
-    "pytest-cov",
     "python-docx",
 ]
 readme = { file = "README.rst", content-type = "text/x-rst" }
 
+[project.optional-dependencies]
+dev = [
+    "pytest",
+    "pytest-cov",
```

Comment:
Could it maybe be an idea to add twine, black, ect. In here also, specifically the ones from the env Makefile command?

Reply:
Black should come with the pre-commit env, but yeah, twine is great candidate! Let me add that!

Reply:
Come to think of it, these:
* pipx install build
* pipx install pre-commit
* pipx install twine

Are preferable to have available stand-alone as do not require senfd to operate. E.g. running code checks etc. directly on the code without building / installing it.. twine should also be able to just upload a given .tar.gz, and of course build needs to be standalone otherwise you end up in a chicken-and-egg problem.